examples/gocean/eg3/Makefile

```diff
@@ -99,8 +98,11 @@ compile-mpi-ocl: dm_fd_lib ${FORTCL_LIB} ${KERNELS} psy_gen_dm.o alg_gen_dm.f90
 %.o: %.f90
 	${F90} ${LDFLAGS} -I${INF_INC} -I${FORTCL_INC} -c $<
 
-compile:
-	echo "This example needs to be compiled with 'make compile-ocl'"
+compile: transform
+	@echo "Example gocean/eg3 needs to be compiled with 'make compile-ocl'"
+
+run: compile
```

Comment:
Should be `compile-ocl`?

Reply:
Hmm - I intentionally did not use ``compile-ocl``, since I assumed that would not work with gfortran - but turns out it does, so I've changed this to ``compile-opencl``.